modules/core/src/main/java/org/apache/ignite/internal/processors/security/GridSecurityProcessor.java

```diff
@@ -77,7 +89,7 @@
     public SecuritySubject authenticatedSubject(UUID subjId) throws IgniteCheckedException;
```

Comment:
Shouldn't we mention that actually not only node subjects can be retrieved here?

Reply:
I thought we will do that in the process of security reworking. 
Currently, these methods don't use to get a security subject of thin clients.